typescript/widgets/src/components/CopyButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {
```

Comment:
Some of these new buttons/icons are actually new but some are code from the Warp UI + Explorer that I'm trying to de-dupe.